<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imglabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Citra Asli</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Citra Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QSlider" name="slider_br">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                stop:0 #B1B1B1, stop:1 #D6D6D6);
    margin: 2px 0;
    border-radius: 4px;
}

/* Active portion of the groove */
QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                stop:0 #6A9CFD, stop:1 #1C6DFF);
    border: 1px solid #1C5FB7;
    height: 8px;
    border-radius: 4px;
}

/* Handle styling */
QSlider::handle:horizontal {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5,
                fx:0.5, fy:0.5, 
                stop:0 #FFFFFF, stop:0.5 #E0E0E0, stop:1 #D0D0D0);
    border: 1px solid #5C5C5C;
    width: 18px;
    height: 18px;
    margin: -6px 0;
    border-radius: 9px;
}

/* Handle when pressed */
QSlider::handle:horizontal:pressed {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5,
                fx:0.5, fy:0.5, 
                stop:0 #F0F0F0, stop:0.5 #D0D0D0, stop:1 #B0B0B0);
    border: 1px solid #3A3A3A;
}

/* Add tick marks */
QSlider::add-page:horizontal {
    background: #E0E0E0;
    border: 1px solid #999999;
    height: 8px;
    border-radius: 4px;
}
</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_br">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_rt">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="zoom_in">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../asset/zoom-in-2-32.ico</normaloff>../asset/zoom-in-2-32.ico</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="zoom_out">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../asset/zoom-out-2-32.ico</normaloff>../asset/zoom-out-2-32.ico</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_reset">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>142</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_reset {
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 14px;
    font-weight: bold;
    border: 2px solid #e74c3c;  
    border-radius: 8px;
  
    color: #e74c3c;  
    background-color: transparent;
    min-width: 30px;
    outline: none;
    margin: 5px;

}


QPushButton#btn_reset:hover {
    background-color: rgba(231, 76, 60, 0.1);
    border: 2px solid #c0392b;  /* Darker red */
    color: #c0392b;
}


QPushButton#btn_reset:pressed {
    background-color: #e74c3c;
    color: white;
    border: 2px solid #e74c3c;
}


QPushButton#btn_reset:disabled {
    border: 2px solid #bdc3c7;
    color: #bdc3c7;
    background-color: transparent;
}
</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-5</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Adjusment</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_clear {
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 14px;
    font-weight: bold;
    border: 2px solid #e74c3c;  
    border-radius: 8px;
  
    color: #e74c3c;  
    background-color: transparent;
    min-width: 30px;
    outline: none;
    margin: 5px;
 
}


QPushButton#btn_clear:hover {
    background-color: rgba(231, 76, 60, 0.1);
    border: 2px solid #c0392b;  
    color: #c0392b;
}


QPushButton#btn_clear:pressed {
    background-color: #e74c3c;
    color: white;
    border: 2px solid #e74c3c;
}


QPushButton#btn_clear:disabled {
    border: 2px solid #bdc3c7;
    color: #bdc3c7;
    background-color: transparent;
}
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufug">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="operasi_titik">
    <property name="title">
     <string>Operasi Citra</string>
    </property>
    <addaction name="actionEqual"/>
    <addaction name="actionGaussian_Filter"/>
   </widget>
   <widget class="QMenu" name="menuKonversi">
    <property name="title">
     <string>Konversi</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBiner"/>
   </widget>
   <addaction name="menufug"/>
   <addaction name="menuKonversi"/>
   <addaction name="operasi_titik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionBiner">
   <property name="text">
    <string>Biner</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionRotasi">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionPenjumlahan">
   <property name="text">
    <string>Penjumlahan</string>
   </property>
  </action>
  <action name="actionEqual">
   <property name="text">
    <string>Equalize Histogram</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show Histogram</string>
   </property>
  </action>
  <action name="actionGaussian_Filter">
   <property name="text">
    <string>Gaussian Filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
